lib/file-helper.js

```diff
@@ -0,0 +1,57 @@
+'use strict'
+
+const fs = require('fs')
+const configPaths = require('../config/paths.json')
+
+// Read component list from various sources
+exports.SrcComponentList = fs.readdirSync(configPaths.src + 'components/')
+exports.DistComponentList = fs.readdirSync(configPaths.dist + 'components/')
+exports.PackagesComponentList = fs.readdirSync(configPaths.packages)
+
+// List all the files for a given component in dist/components
+//
+// This helper function takes a component name and returns the corresponding
+// files found in the dist/components folder of that component
+const distFilesForComponent = componentName => {
+  return fs.readdirSync(configPaths.dist + 'components/' + componentName)
+}
+exports.distFilesForComponent = distFilesForComponent
+
+// List all expected files for a given component in dist/components
+//
+// This helper function takes a component name and returns the expected files
+// to be found in the dist/components folder of that component
+const expectedDistFilesForComponent = componentName => {
+  // We don't want to include the yaml file and index.njk in dist
+  let srcComponentFiles = fs.readdirSync(configPaths.src + 'components/' + componentName)
+                        .filter(file => (file !== componentName + '.yaml' && file !== 'index.njk'))
+  // We expect dist to contain an html sample
+  srcComponentFiles.push(componentName + '.html')
+  return srcComponentFiles.sort()
```

Comment:
We're sorting the files here and in `tasks.spec.js`. I wonder if sorting is needed as `to.have.same.members` in Chai doesn't seem to check order, there's a separate `to.have.ordered.members`.

Reply:
There is no need to sort the files when looking for the same members, but it makes it a lot easier to figure the differences between ` + expected` and `- actual` when tests are failing.

Reply:
Aha that makes sense. I've left a comment below in the code to explain this.